apps/heft/src/plugins/TypeScriptPlugin/TypeScriptBuilder.ts

```diff
@@ -207,16 +207,26 @@ export class TypeScriptBuilder extends SubprocessRunnerBase<ITypeScriptBuilderCo
     }
 
     this._tslintConfigFilePath = path.resolve(this._configuration.buildFolder, 'tslint.json');
-    this._eslintConfigFilePath = path.resolve(this._configuration.buildFolder, '.eslintrc.js');
     this._eslintEnabled = this._tslintEnabled =
       this._configuration.lintingEnabled && !this._configuration.watchMode; // Don't run lint in watch mode
 
     if (this._tslintEnabled) {
       this._tslintEnabled = this._cachedFileSystem.exists(this._tslintConfigFilePath);
     }
 
+    this._eslintConfigFilePath = path.resolve(this._configuration.buildFolder, '.eslintrc.js');
```

Comment:
I'd suggest factoring this out into a `findESLintConfig` helper function and not repeatedly assign the value.

Reply:
Good suggestion. Extracted to a `_resolveEslintConfigFilePath()` method.